tests/test_core.py

```diff
@@ -8709,6 +8750,8 @@ def setUp(self):
 wasm3 = make_run('wasm3', emcc_args=['-O3'])
 wasms = make_run('wasms', emcc_args=['-Os'])
 wasmz = make_run('wasmz', emcc_args=['-Oz'])
+# wasm64 = make_run('wasm64', emcc_args=['-O0', '-g3', '-s', 'MEMORY64=2'],
+#                   settings=None, env=None, wasm64=True)
```

Comment:
Don't you want to add this?

You can do `settings={'MEMORY64':2}` instead of adding to `emcc_args`.